### ISSUE TITLE
code to reproduce the ImageNet results

### ISSUE THREAD

**Issue by chaithu536** [google-research/uda#25]
I enjoyed reading the paper and very promising results. Are you planning to share the code of ImageNet experiments in semi-supervised setting ? It would be a great help to reproduce the results.

**Comment by qizhex** [comment]
Thanks! We don't plan to release the ImageNet code because it requires a lot of code cleaning. The hyperparameters for ImageNet experiments are available [here](https://github.com/google-research/uda/issues/5). The augmentation policy is available [here](https://github.com/google-research/uda/blob/master/image/autoaugment/policies.py#L178).

**Comment by chaithu536** [comment]
Thanks for the links. Is it possible to share the ImageNet trained model or weights ? How did you choose the 10% labelled data from ImageNet ? Did you pick random images from each class ?

**Comment by qizhex** [comment]
Hi, we used random images from the whole ImageNet and didn't keep the class balanced. We used this [script](https://github.com/tensorflow/models/blob/master/research/inception/inception/data/build_imagenet_data.py) to preprocess the ImageNet and saved the data into 1024 files. We used the first 102 files as the training data. Tensorflow is a bit different from pytorch. If we only share the model weights without the code, it cannot be executed.

**Comment by chaithu536** [comment]
Thank you for providing link to the script. I just want to confirm that the data from only these 102 files are used for fully supervised learning to obtain 55.09% Top-1 accuracy as reported in the paper (Table 3). Is this correct ? Which network architecture is used to train on this dataset ?

**Comment by qizhex** [comment]
Yes, that's correct. We use ResNet-50 as our baseline model. The code of our baseline was obtained [here](https://github.com/tensorflow/tpu/tree/master/models/official/resnet).
